examples/django_chat/sdjango/sd_manager.py

```diff
@@ -0,0 +1,22 @@
+from socketio.base_manager import BaseManager
+
+
+class SdManager(BaseManager):
```

Comment:
yes, this can be removed. Originally, I plan to do some glue code for django here. However, it seems that nothing can do here.